tools/integration-test/src/tests/fee/non_fee_channel.rs

```diff
@@ -0,0 +1,100 @@
+use ibc_test_framework::prelude::*;
```

Comment:
@ljoss17 Does this sound like an accurate description of this test in light of your answer? 🙂

```suggestion
//! This test ensures that the relayer defaults the `channel_version` to `ics20`, the expected 
//! behavior when fees are not enabled, when the fee transaction is not configured correctly. 
//! In the case of this test, the call to `register_counterparty_payee` returns an error because 
//! the `channel_version` override to set the version to `ics20_with_fee` is left out. The test
//! ensures then that the transaction still follows through with the expected `ics20` behavior.

use ibc_test_framework::prelude::*;
```

Reply:
The `ibc_token_transfer_with_fee` will also fail as it contains the fees. I think would be nice if it specified that both will return an error, so the transfer without fee needs to be used in order to transfer tokens. Something along the lines of:
```
//! In the case of this test, the calls to `register_counterparty_payee` and `ibc_token_transfer_with_fee`
//! return an error because the `channel_version` override to set the version to `ics20_with_fee` is left out.
//! The test ensures then that the transaction still follows through if the transfer without fees is used, `ibc_transfer_token`.
```